src/test/rustdoc-ui/track-diagnostics.rs

```diff
@@ -0,0 +1,10 @@
+// compile-flags: -Z track-diagnostics
+// error-pattern: created at
+
+// Normalize the emitted location so this doesn't need
+// updating everytime someone adds or removes a line.
+// normalize-stderr-test ".rs:\d+:\d+" -> ".rs:$$LINE::$$COL"
```

Comment:
Unimportant, but this should probably be 
```suggestion
// normalize-stderr-test ".rs:\d+:\d+" -> ".rs:$$LINE:$$COL"
```
Or maybe even
```suggestion
// normalize-stderr-test ".rs:\d+:\d+" -> ".rs:LL:CC"
```
(`LL` is used on the left of diagnostics as you can notice)
(and similarly in other tests)